### ISSUE TITLE
Division by zero

### ISSUE THREAD

**Issue by diekotto** [thorie7912/bitcoin-calculator#7]
The production page are actually showing an error on trying to divide by zero and the calculations are not made.
https://calculator.for-bitcoin.com/

**Comment by diekotto** [comment]
Seems like the api are not fetching the price.

**Comment by microlancer** [comment]
Good catch thanks for letting me know! I'll try to fix this asap

**Comment by microlancer** [comment]
Looks like the coindesk API is missing dates:

https://api.coindesk.com/v1/bpi/historical/close.json?start=2020-08-14&end=2020-08-24

```
{
  "bpi": {
    "2020-08-14": 11759.5,
    "2020-08-16": 11908.33,
    "2020-08-17": 11856.5,
    "2020-08-18": 12327.5,
    "2020-08-19": 11918.5,
    "2020-08-23": 11611.5
  },
  "disclaimer": "This data was produced from the CoinDesk Bitcoin Price Index. BPI value data returned as USD.",
  "time": {
    "updated": "Aug 25, 2020 00:03:00 UTC",
    "updatedISO": "2020-08-25T00:03:00+00:00"
  }
}
```

The output is missing 8/15, 8/20, 8/21, 8/22, 8/23, 8/24.

I'll see if I can just use the previous date prices as placeholders.

**Comment by microlancer** [comment]
Fixed in commit dd2533caeeb0128a758a73e460c24c1ebd1d60b3
